lib/chromic_pdf/pdf/protocol_macros.ex

```diff
@@ -235,22 +232,50 @@ defmodule ChromicPDF.ProtocolMacros do
     with true <- JsonRPC.notification?(msg, "Runtime.exceptionThrown"),
          true <- state["sessionId"] == msg["sessionId"] do
       exception = get_in!(msg, ["params", "exceptionDetails"])
+      description = get_in!(exception, ["exception", "description"])
 
       case Map.get(state, :unhandled_runtime_exceptions, :log) do
         :ignore ->
-          :no_match
+          {:match, :keep, state}
```

Comment:
Why does this no longer return `:no_match`?  What is the purpose of this change?

I am confused, as this will lead in the with clause of line 159 to not match anymore and thus return `{:match, :keep, state}` instead of running the block. Or am I missing something?

Reply:
Yes, I reconsidered this code while working on the (very similar) logic for `console_api_calls`: Figured that even though we ignore the exception, it doesn't really make sense to match it against the actual awaited message pattern. Or does it? Couldn't fathom a scenario where I'd wait for an exception to be thrown before continueing in a protocol.